task/summary/0.1/summary.yaml

```diff
@@ -45,5 +51,18 @@ spec:
         if [ "$BUILD_TASK_STATUS" == "Succeeded" ]; then
           echo "Generated Image is in : $IMAGE_URL"
         fi
+        if [ -e "$SOURCE_BUILD_RESULT_FILE" ]; then
+          found=
+          fields=($(cat "$SOURCE_BUILD_RESULT_FILE"))
+          for f in ${fields[@]}; do
+            if [ -n "$found" ]; then
+              echo "Generated Source Image is in : $(echo $f | tr -d '",')"
+              break
+            fi
+            if [ "$f" == '"image_url":' ]; then
+              found=yes
+            fi
+          done
+        fi
```

Comment:
Instead of parsing JSON in Bash, can the source-build task write something more easily parse-able to the workspace file? Maybe just the image url?

Reply:
I was intended to make the result file be general for sharing with other tasks, not only for summary even though only summary uses for now.

Reply:
Oh, you remind me that we can use https://github.com/redhat-appstudio/build-tasks-dockerfiles/tree/main/task-toolset @mmorhun

Reply:
> make the result file be general for sharing with other tasks

IMO that's generally a bad idea. Files in the workspace can't really be trusted, since the user can always add a custom task that modifies the workspace content. That's also why the EC team is doing the Trusted Artifacts work #791

Reply:
(although for the show-summary task this is fine, it's purpose is just to inform the user)

Reply:
@chmeliik PTAL https://github.com/redhat-appstudio/build-definitions/pull/857